src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameUserInputComboBox.xaml

```diff
@@ -87,7 +87,6 @@
                     </Trigger>
                     <Trigger Property="IsKeyboardFocused" Value="true">
                         <Setter Property="BorderBrush" Value="{DynamicResource {x:Static vsui:CommonControlsColors.ButtonBorderFocusedBrushKey}}"/>
-                        <Setter Property="Background" Value="{DynamicResource {x:Static vsui:CommonControlsColors.ButtonFocusedBrushKey}}"/>
```

Comment:
Why this is not needed?

Reply:
We want to make it consistent with the panel expand case (Purple), and different with the panel collapse case. It decreases the confusion. 

After removing the focus background, the button will be purple when focused.  And the light border can indicate whether the button is focused.

Reply:
Could you post a screenshot before & after?

Reply:
Could you verify if the color still looks correct when in light/blue theme?